python/lsst/meas/base/base.py

```diff
@@ -262,6 +263,23 @@ def fail(self, measRecord, error=None):
                    % self.__class__.__name__)
         raise NotImplementedError(message)
 
+    @staticmethod
+    def getTransformClass():
+        """!
+        Get the measurement transformation appropriate to this plugin.
+
+        This returns subclass of MeasurementTransform, which may be
+        instantiated with details of the algorithm configuration and then
+        called with information about calibration and WCS to convert from raw
+        measurement quantities to calibrated units. Calibrated data is then
+        provided in a separate output table.
+
+        By default, we assume that no transformation is appropriate, and
+        therefore return NullTransform.
+        """
```

Comment:
I think that PassThroughTransform might be a less surprising default - or is the expectation that by default you don't want to present users with uncalibrated data, and so all such should be removed prior to database ingest?